internal/backends/winit/Cargo.toml

```diff
@@ -64,11 +65,12 @@ rgb = { version = "0.8.27", optional = true }
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 web-sys = { version = "0.3", features=["HtmlInputElement", "HtmlCanvasElement", "Window", "Document", "Event", "KeyboardEvent", "InputEvent", "CompositionEvent", "DomStringMap", "ClipboardEvent", "DataTransfer"] }
 wasm-bindgen = { version = "0.2" }
-send_wrapper = { workspace = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 glutin = { version = "0.30", optional = true, default-features = false, features = ["egl", "wgl"] }
 glutin-winit = { version = "0.3.0", optional = true, default-features = false, features = ["egl", "wgl"] }
+accesskit = { version = "0.11.0" }
+accesskit_winit = { version = "0.14.0" }
```

Comment:
In my opinion it should not be optional. Afaics it comes at no runtime cost until it is used. It does come at a compile time cost though. But by the time an end-user discovers that the application is not accessible it is IMO too late.

Reply:
Another option: we could add it do the default features, so it's possible to opt out of it. How about that?

Reply:
Because of how heavy accessibility support is on Linux, a problem for which I haven't yet found a good solution, having a way to opt out is acceptable to me. I'm glad it's enabled by default.